detail/provisioning.cattle.io.cluster.vue

```diff
@@ -286,6 +286,10 @@ export default {
 
       return false;
     },
+
+    isClusterActive() {
+      return this.value.state === 'active';
+    }
```

Comment:
Active is not quite the right state.. it can be active but not connected or not active (e.g. updating) but still snapshot-able.

What you want is whether the cluster is "ready", which means it's connected and can be operated on.  `this.value.mgmt?.isReady`.

Reply:
Got it, thanks!

I've updated both PRs for this change. Just to make sure I understand correctly, we should be able to snapshot when the cluster is "ready" but has a state of "provisioning", correct?

![image](https://user-images.githubusercontent.com/835961/127930082-517015ab-63d0-4627-ab36-34c12a71a0dd.png)